common/component_interface_utils/include/component_interface_utils/rclcpp/service_server.hpp

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef COMPONENT_INTERFACE_UTILS__RCLCPP__SERVICE_SERVER_HPP_
+#define COMPONENT_INTERFACE_UTILS__RCLCPP__SERVICE_SERVER_HPP_
+
+#include <rclcpp/rclcpp.hpp>
+
+namespace component_interface_utils
+{
+
+/// The wrapper class of rclcpp::Service for logging.
+template <class SpecT>
+class Service
+{
+public:
+  RCLCPP_SMART_PTR_DEFINITIONS(Service)
+
+  template <class NodeT>
+  using CallbackType = void (NodeT::*)(
+    typename SpecT::Service::Request::SharedPtr, typename SpecT::Service::Response::SharedPtr);
+
+  /// Constructor.
+  explicit Service(typename rclcpp::Service<typename SpecT::Service>::SharedPtr service)
+  {
+    service_ = service;
```

Comment:
For what do you store `service_`? :thinking: 
It seems not used, right?

Reply:
No, it's not be used. But it's necessary to maintain the reference count of shared pointer.

Reply:
I see, then I think it's good to keep a comment. Could you write it?